js/app/components/AddressAutosuggest.js

```diff
@@ -246,9 +246,7 @@ const generic = {
       <span></span>
     )
   },
-  highlightFirstSuggestion: function() {
-    return false
-  },
```

Comment:
Does this break anything? If not, I'd keep it just in case

Reply:
there is a warning in the console (you can see it on master or on demon instance), which say that it should be a boolean, so i modified it ( see here [https://github.com/moroshko/react-autosuggest](https://github.com/moroshko/react-autosuggest) )